test/unit/loggingManagerTests.js

```diff
@@ -287,18 +333,19 @@ describe('LoggingManager', function () {
     })
   })
 
-  describe('checkLogLevel', function () {
-    it('should request log level override from the config map', function () {
-      this.logger.checkLogLevel()
-      this.Fs.readFile.should.have.been.calledWithMatch(
+  describe('checkLogLevelFile', function () {
+
+    it('should request log level override from the config map', async function () {
+      await this.logger.checkLogLevelFile()
+      this.Fs.promises.readFile.should.have.been.calledWithMatch(
         '/logging/tracingEndTime'
       )
     })
 
     describe('when read errors', function () {
       beforeEach(function () {
-        this.Fs.readFile.yields(new Error('error'))
-        this.logger.checkLogLevel()
+        this.Fs.promises.readFile.yields(new Error('error'))
```

Comment:
Let's change the stubs to look more like promises:
```suggestion
        this.Fs.promises.readFile.throws(new Error('error'))
```